src/packages.builds

```diff
@@ -8,14 +8,37 @@
     <BuildAllOSGroups Condition="'$(FilterToOSGroup)' != ''">false</BuildAllOSGroups>
   </PropertyGroup>
 
+  <!--
+    the following packages are built and shipped from TFS, so while we still want
+    to build them we don't want to ship them.  by setting NonShippingPackage to true
+    the packages will still be built but placed in a different location.
+  -->
   <ItemGroup>
-    <Project Include="Native\pkg\**\*.builds" Condition="'$(SkipNativePackageBuild)' != 'true'">
+    <NonShippingPackage Include="Native\pkg\runtime.native.System.Data.SqlClient.sni\*.builds" Condition="'$(SkipNativePackageBuild)' != 'true'">
+      <AdditionalProperties>$(AdditionalProperties);SkipCreatePackageOnMissingFiles=true;NonShippingPackage=true</AdditionalProperties>
+      <BuildAllOSGroups>$(BuildAllOSGroups)</BuildAllOSGroups>
+    </NonShippingPackage>
+    <NonShippingPackage Include="Native\pkg\runtime.native.System.IO.Compression\*.builds" Condition="'$(SkipNativePackageBuild)' != 'true'">
+      <AdditionalProperties>$(AdditionalProperties);SkipCreatePackageOnMissingFiles=true;NonShippingPackage=true</AdditionalProperties>
+      <BuildAllOSGroups>$(BuildAllOSGroups)</BuildAllOSGroups>
+    </NonShippingPackage>
+    <NonShippingPackage Include="System.Data.SqlClient\pkg\*.builds" Condition="'$(SkipManagedPackageBuild)' != 'true'">
+      <AdditionalProperties>$(AdditionalProperties);NonShippingPackage=true</AdditionalProperties>
+    </NonShippingPackage>
+    <NonShippingPackage Include="System.IO.Compression\pkg\*.builds" Condition="'$(SkipManagedPackageBuild)' != 'true'">
+      <AdditionalProperties>$(AdditionalProperties);NonShippingPackage=true</AdditionalProperties>
+    </NonShippingPackage>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Project Include="Native\pkg\**\*.builds" Exclude="@(NonShippingPackage)" Condition="'$(SkipNativePackageBuild)' != 'true'">
       <AdditionalProperties>$(AdditionalProperties);SkipCreatePackageOnMissingFiles=true</AdditionalProperties>
       <BuildAllOSGroups>$(BuildAllOSGroups)</BuildAllOSGroups>
     </Project>
-    <Project Include="*\pkg\*.builds" Condition="'$(SkipManagedPackageBuild)' != 'true'">
+    <Project Include="*\pkg\*.builds" Exclude="@(NonShippingPackage)" Condition="'$(SkipManagedPackageBuild)' != 'true'">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
+    <Project Include="@(NonShippingPackage)" />
```

Comment:
I would add the NonShippingPackage=true metadata here instead of duplicating it in all the above locations.